src/Homework_02/Dop_Palindrom.py

```diff
@@ -0,0 +1,28 @@
+'''
+Определите, является ли число палиндромом
+(читается слева направо и справа налево одинаково).
+Число положительное целое, произвольной длины.
+Задача требует работать только с числами
+(без конвертации числа в строку)
+'''
+
+num = int(input('Введите число\n'))
+num_length = 1
+num_division = num
+while num_division // 10 > 0:
+    num_length += 1
+    num_division //= 10
+print('Длина числа', num_length)
+
+num_x = num
+for i in range(num_length // 2):
```

Comment:
can you just revert number using arithmetic operators?

Reply:
Done. It's even easier)